File: C013_WebClient.js
const https = require('https');

https.get('https://jsonplaceholder.typicode.com/todos/1', (response) => {
  console.log(response.statusCode + (response.statusCode==200? " OK" : " Failed"));

  let data = '';
  response.on('data', (chunk) => {
    data += chunk;
  });

  response.on('end', () => {
    console.log(JSON.stringify(JSON.parse(data), null, 2));
  });
});

File: C013_WebClient.py
#!/usr/bin/env pythong36

import http.client
import json

conn = http.client.HTTPSConnection("jsonplaceholder.typicode.com")
headers = {"Content-type": "application/json"}
conn.request("GET", "/todos/1", None, headers)
response = conn.getresponse()
print(response.status, response.reason)

data = json.dumps(json.loads(response.read()), indent=2)
print(data)
conn.close()

